test/check-pages

```diff
@@ -49,8 +49,10 @@ WantedBy=default.target
         b.enter_page("/system/init")
         b.wait_text("#service-unit .panel-heading", "Test Service")
 
-        # check that the login dance preserves parameters
-        m.execute("systemctl restart cockpit-testing.service")
+        # testing service has unmet dependencies on atomic
+        if not 'atomic' in m.os:
+            # check that the login dance preserves parameters
+            m.execute("systemctl restart cockpit-testing.service")
```

Comment:
`cockpit-testing.servce` is just a variant of `cockpit.service`, it does not start the test-server, so it shouldn't have dependency problems.